locale/en/foundation/case-studies.md

```diff
@@ -18,6 +18,11 @@ preferences (current and future) for front end, back end and IoT developers.
 * [See the infographic.](/static/documents/2016-survey-infographic.png)
 * [Read the full report.](/static/documents/2016-survey-report.pdf)
 
+## Presentation - The Enterprise Case for Node.js
+The Node.js Foundation has consolidated all the best thinking around why enterprises should consider Node.js for their systems of engagement in this short presentation.
+
+<iframe src="//www.slideshare.net/slideshow/embed_code/key/cdqy9ghQ1Bl48A" width="595" height="485" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; margin-bottom:5px; max-width: 100%;" allowfullscreen> </iframe> <div style="margin-bottom:5px"> <strong> <a href="//www.slideshare.net/NodejsFoundation/the-enterprise-case-for-nodejs-65523959" title="The Enterprise Case for Node.js" target="_blank">The Enterprise Case for Node.js</a> </strong> from <strong><a target="_blank" href="//www.slideshare.net/NodejsFoundation">NodejsFoundation</a></strong> </div>
```

Comment:
@gtewallace it is probably sufficient to only add the `<iframe>`:
```html
<iframe width="560" height="400" src="//www.slideshare.net/slideshow/embed_code/key/cdqy9ghQ1Bl48A" frameborder="0" allowfullscreen></iframe></body>
</html>
```
unless the caption is also required.